items/test_models.py

```diff
@@ -0,0 +1,46 @@
+from django.test import TestCase
+from .models import Item, Group
+from django.contrib.auth.models import User
+from django.utils import timezone
+from django.db import IntegrityError
+
+
+class ItemModelTest(TestCase):
```

Comment:
This tests writed by Nikita.This file appeared after squash.It already in master branch.